src/pretix/plugins/socialauth/views.py

```diff
@@ -28,3 +47,38 @@ def oauth_return(request):
         )
         logger.error('Error while authorizing: user has no email address.')
         return redirect('control:auth.login')
+
+
+class LoginState(Enum):
```

Comment:
Please subclass from [`StrEnum`](https://docs.python.org/3.11/library/enum.html#enum.StrEnum). By that, you can compare directly with `str`, like `v === LoginState.ENABLED`.